dashboard.json

```
@@ -3329,8 +3329,8 @@
   },
   "timepicker": {},
   "timezone": "browser",
-  "title": "KeyCloak MicroProfile Metrics",
-  "uid": null,
+  "title": "Keycloak",
+  "uid": "keycloak",
```

This change would prevent the uid's from being generated by Grafana and it would therefore not be possible to monitor multiple KeyCloak instances with one Grafana instance. You can use the Grafana API to get the current UID of the Dashboard or to automatically generate shortened links. You can have a look at [one](https://github.com/ZPascal/grafana_api_sdk) of my other projects to solve your problem.

```suggestion
  "uid": null,
```

---

The dashboard has variables to switch between `job` and `instance`. I think using them is the right method to monitor multiple. WDYT? Also, when adding a dashboard with a predefined uid, Grafana allows to update the uid on import time.